src/Auth_Command.php

```diff
@@ -730,5 +740,41 @@ public function list( $args, $assoc_args ) {
 			}
 		}
 	}
+
+	/**
+	 * This will display all the auths of all the sites
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	private function display_all_auths() {
+
+		// Fetch all the auths across all the sites.
+		$sites = Auth::all();
+
+		$formatter = new EE\Formatter( $assoc_args, [ 'sitename', 'username', 'password' ] );
+
+		// Add a field named 'sitename' to the array , so that it can be displayed as heading in output.
+		$result = array_map(
+			function ( $site ) {
+				$site->sitename = $site->site_url;
+				return $site;
+			},
+			$sites
+		);
+
+		// Sorts the $result array to make sure that same sites appear together.
+		usort(
+			$result,
+			function ( $item1, $item2 ) {
+
+				return strcmp( $item1->sitename, $item2->sitename );
```

Comment:
Remove extra spaces from above and below. Also run PHPCS and fix errors only the places where you have edited he code.

Reply:
@kirtangajjar  Removed the extra spaces and fixed the phpcs issues using the `phpcs.xml.dist` file